plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveConnectorTest.java

```diff
@@ -3704,7 +3722,7 @@ public void testInsertTableWithHeaderAndFooterForCsv()
                         ")\n" +
                         "WITH (\n" +
                         "   format = 'CSV',\n" +
-                        "   skip_header_line_count = 1\n" +
+                        "   skip_header_line_count = 12\n" +
```

Comment:
use `2` to be as near to `1` as possible (to catch possible issues with misuse of `>`/`>=`....)

Reply:
ok